### ISSUE TITLE
[11 alpha04] User name visibility

### ISSUE THREAD

**Issue by Valdnet** [nextcloud/talk-android#993]
Steps:
1. Choose dark theme.
2. Go to the list of users for conversations.
3. Choose a bright theme.
4. Go to the list of users for conversations.

The color of the user names or groups do not change immediately. Only after restarting the application.

![obraz](https://user-images.githubusercontent.com/47037905/106641516-97668200-6587-11eb-98c8-aa457248138f.png)
App:
Android 10
Nextcloud Talk 11.0.0 Alpha 04

**Comment by Chartman123** [comment]
Possible duplicate of #915

**Comment by mahibi** [comment]
duplicate of #915
